README.md

```diff
@@ -10,7 +10,7 @@ Now you can use a simple and pure javascript lib to mask your input elements, wi
 Let's live a lightweight client-side world using micro libraries as VanillaMasker is!
 Don't worry about where this will work, because this is a cross-browser and cross-device library. And if you find any bug, please let us know about it [reporting an issue](https://github.com/Creditas/vanilla-masker/issues).
 
-If you wanna see how this lib works, click to test this [demo page](http://bankfacil.github.io/vanilla-masker/demo.html).
```

Comment:
For me it doesn't load the styles when trying to hit the `demo.html`:

![image](https://cloud.githubusercontent.com/assets/2780941/23102891/47f859d8-f690-11e6-81bd-ae2aa057fcc4.png)